composer.json

```diff
@@ -113,10 +113,16 @@
             "drupal/panels": {
                 "2878684 - Use String.match to correlate regions when switching Layouts in Panels IPE":
                 "https://www.drupal.org/files/issues/panels-ipe-2878684-3.patch",
-                "2878650 - Improve Layout Picker styling in Panels IPE":
-                "https://www.drupal.org/files/issues/panels-ipe-layout-picker-styling.patch",
                 "2825034 - Form error messages do not appear in IPE":
                 "https://www.drupal.org/files/issues/panels-ipe-propogate-errors-2825034-5.patch"
+            },
+            "drupal/ctools": {
```

Comment:
All of these CTools patches have landed, and are necessary for inline blocks to work.